apps/browser/qml/pages/PermissionExceptionsPage.qml

```diff
@@ -92,7 +96,24 @@ Page {
                 MenuItem {
                     //% "Delete"
                     text: qsTrId("sailfish_browser-me-delete")
-                    onClicked: remove(model.index)
+                    onClicked: remove(model.uri, model.type, model.capability)
+                }
+            }
+        }
+
+        PullDownMenu {
+            visible: model && (view.count > 0)
```

Comment:
Sorry I didn't realize that you already had pull down menu... now there two of them.